.github/styles/Vocab/Docs/accept.txt

```diff
@@ -29,6 +30,7 @@ Kafka
 Kaptain
 Katib
 Keras
+keyspace
 Kibana
 Knative
 Kommander
```

Comment:
Can we also add `Kubecost` in this list?

Reply:
also `kubeconfig`

Reply:
I added them.